robottelo/ui/locators/menu.py

```diff
@@ -4,6 +4,16 @@
 from selenium.webdriver.common.by import By
 from .model import LocatorDict
 
+# This is the old menu container xpath, was failing in upstream
+# kept here for reference
+# MENU_CONTAINER_XPATH = (
+#     "//div["contains(@style,'static') or contains(@style,'fixed')]"
+# )
+
+# Downstream has 2 elements with id=menu we should get last()
```

Comment:
I would keep away from 'downstream' term in the code as it is too application specific